src/applications/vaos/tests/appointment-list/components/CanceledAppointmentsList.unit.spec.js

```diff
@@ -1,62 +1,80 @@
-import { mockFacilitiesFetchByVersion } from '../../mocks/fetch';
-import {
-  createMockAppointmentByVersion,
-  createMockFacilityByVersion,
-} from '../../mocks/data';
+import { renderWithStoreAndRouter } from '../../mocks/setup';
+import { getVAOSRequestMock } from '../../mocks/v2';
+import { mockVAOSAppointmentsFetch } from '../../mocks/helpers.v2';
 import { APPOINTMENT_STATUS, VIDEO_TYPES } from '../../../utils/constants';
+import { createMockFacilityByVersion } from '../../mocks/data';
 
-const initialState = {
-  featureToggles: {
-    vaOnlineSchedulingCancel: true,
-  },
-};
-
-describe.skip('VAOS Component: CanceledAppointmentsList', () => {
+describe('VAOS Component: CanceledAppointmentsList', () => {
   beforeEach(() => {
     mockFetch();
-    MockDate.set(getTestDate());
-    mockFacilitiesFetchByVersion({ version: 0 });
-  });
-  afterEach(() => {
-    MockDate.reset();
   });
-  it('should show information without facility name', async () => {
-    const startDate = moment.utc();
-    const data = {
+
+  const initialState = {
+    featureToggles: {
+      vaOnlineSchedulingCancel: true,
+      vaOnlineSchedulingVAOSServiceVAAppointments: true,
+    },
+  };
+
+  // This function creates a baseline canceled appointment
+  function mockBaseCanceledAppointment(start) {
+    const appointment = getVAOSRequestMock();
+    appointment.id = '1234';
+    appointment.attributes = {
       id: '1234',
       status: APPOINTMENT_STATUS.cancelled,
       kind: 'clinic',
       clinic: '308',
-      start: startDate.format(),
+      localStartTime: start.format('YYYY-MM-DDTHH:mm:ss.000ZZ'),
       locationId: '983GC',
     };
-    const appointment = createMockAppointmentByVersion({
-      version: 0,
-      ...data,
+    return appointment;
+  }
+
+  // This function mocks the behavior of fetchFutureAppointments
+  // which fetches appointments and appointment requests
+  function mockFetchFutureAppointments(appointment) {
+    mockVAOSAppointmentsFetch({
+      start: moment()
+        .subtract(30, 'days')
+        .format('YYYY-MM-DD'),
+      end: moment()
+        .add(395, 'days')
+        .format('YYYY-MM-DD'),
+      statuses: ['booked', 'arrived', 'fulfilled', 'cancelled'],
+      requests: [appointment],
+    });
+    mockVAOSAppointmentsFetch({
+      start: moment()
+        .subtract(120, 'days')
+        .format('YYYY-MM-DD'),
+      end: moment().format('YYYY-MM-DD'),
+      statuses: ['proposed', 'cancelled'],
+      requests: [appointment],
     });
+  }
+
+  it('should show information without facility name', async () => {
+    // Arrange
+    const startDate = moment();
+    const appointment = mockBaseCanceledAppointment(startDate);
+    mockFetchFutureAppointments(appointment);
 
-    mockAppointmentInfo({ va: [appointment] });
+    // Act
     const screen = renderWithStoreAndRouter(<CanceledAppointmentsList />, {
       initialState,
-      reducers,
```

Comment:
Not sure what the functions were for these reducers but the tests are passing without them so I removed them.